### ISSUE TITLE
Mixed Content warning when viewing racks with images

### ISSUE THREAD

**Issue by XL-Reaper** [netbox-community/netbox-docker#292]
## Current Behavior

When opening the rack view, for instance https://netbox.domain.com/dcim/racks/1/ and the rack has devices which contain images for the front view, the browser (Google Chrome Version 81.0.4044.138) generates this warning:

Mixed Content: The page at 'https://netbox.domain.com/dcim/racks/1/' was loaded over HTTPS, but requested an insecure image 'http://netbox.domain.com/media/devicetype-images/patchpanel.jpg'. This content should also be served over HTTPS.

## Expected Behavior

The page should load fine without this warning.
The images should be loaded over https as well.

# Debug Information

## docker-compose version
docker-compose version 1.25.0, build unknown
docker-py version: 4.1.0
CPython version: 3.8.2
OpenSSL version: OpenSSL 1.1.1f  31 Mar 2020

##  docker version
Client:
 Version:           19.03.8
 API version:       1.40
 Go version:        go1.13.8
 Git commit:        afacb8b7f0
 Built:             Wed Mar 11 23:42:35 2020
 OS/Arch:           linux/amd64
 Experimental:      false

Server:
 Engine:
  Version:          19.03.8
  API version:      1.40 (minimum version 1.12)
  Go version:       go1.13.8
  Git commit:       afacb8b7f0
  Built:            Wed Mar 11 22:48:33 2020
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.3.3-0ubuntu2
  GitCommit:        
 runc:
  Version:          spec: 1.0.1-dev
  GitCommit:        
 docker-init:
  Version:          0.18.0
  GitCommit:        

## git rev-parse HEAD 
5769684c98efaa66049189793e66998b217c5ffa

## The command you used to start the project
docker-compose up -d

## docker inspect netboxcommunity/netbox:latest-ldap --format "{{json .Config.Labels}}"

```json
{
   "NETBOX_GIT_BRANCH":"",
   "NETBOX_GIT_REF":"",
   "NETBOX_GIT_URL":"",
   "ORIGINAL_TAG":"",
   "org.label-schema.build-date":"",
   "org.label-schema.description":"A container based distribution of Netbox, the free and open IPAM and DCIM solution.",
   "org.label-schema.name":"Netbox Docker",
   "org.label-schema.schema-version":"1.0",
   "org.label-schema.url":"https://github.com/netbox-community/netbox-docker",
   "org.label-schema.usage":"https://github.com/netbox-community/netbox-docker/wiki",
   "org.label-schema.vcs-ref":"",
   "org.label-schema.vcs-url":"https://github.com/netbox-community/netbox-docker.git",
   "org.label-schema.vendor":"The netbox-docker contributors.",
   "org.label-schema.version":"snapshot",
   "org.opencontainers.image.authors":"The netbox-docker contributors.",
   "org.opencontainers.image.created":"",
   "org.opencontainers.image.description":"A container based distribution of Netbox, the free and open IPAM and DCIM solution.",
   "org.opencontainers.image.documentation":"https://github.com/netbox-community/netbox-docker/wiki",
   "org.opencontainers.image.licenses":"Apache-2.0",
   "org.opencontainers.image.revision":"",
   "org.opencontainers.image.source":"https://github.com/netbox-community/netbox-docker.git",
   "org.opencontainers.image.title":"Netbox Docker",
   "org.opencontainers.image.url":"https://github.com/netbox-community/netbox-docker",
   "org.opencontainers.image.vendor":"The netbox-docker contributors.",
   "org.opencontainers.image.version":"snapshot"
}

**Comment by XL-Reaper** [comment]
The device type view loads the image fine and without warning by the way, for instance:
https://netbox.domain.com/dcim/device-types/1/

**Comment by cimnine** [comment]
Thanks for bringing it to our attention.
Currently I don't have time to debug unfortunately. But I suspect either our Nginx configuration or an upstream issue to be the cause of this.

**Comment by cimnine** [comment]
So far I can only confirm this issue.
I believe it can be fixed using https://docs.djangoproject.com/en/dev/ref/settings/#std:setting-SECURE_PROXY_SSL_HEADER.
But I was not successful so far.

**Comment by cimnine** [comment]
I believe this issue is resolved since #308 is merged, isn't it?

**Comment by XL-Reaper** [comment]
Yes, no more warning now. Thanks :)
